CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing to the Self Hosted Show Wiki
+
+
+## Objectives of This Wiki
+
+In order to contribute well, you need to understand what the objectives are behind this wiki.
+
+What we are trying to create:
+- A compilation of tools, tips and tricks for self-hosting specific applications.
+- A place where beginners and experts can learn about how to quickly, efficiently and securely deploy applications on their own server(s).
+- Snippets that people can quickly reference in deploying applications.
+- A hub for quality documentation (to the extent possible).
+
+What we are not trying to do:
+- Be another large list of self-hosted tools. See [awesome-selfhosted](https://github.com/awesome-selfhosted/awesome-selfhosted) for a comprehensive list of self-hosted applications.
+- Be a hub for script kitties. People should _learn_ from documentation here not just come here to copy/paste. Snippets are important but should be documented properly.
+- Insecure, sloppy or any undesirable systems administration practices such as:
+  - Executing remote code by `curl`-ing an untrusted script and piping it to `sudo bash`: `curl https://example.com/awesomescript.sh | sudo bash`
+  - Installing additional bloat within a Docker container
+  - Etc.
+
+
+We want to encourage the use of dev/ops tools such as Docker, Ansible, SALT Stack, etc. for increased ease of use and for automating/standardizing as much as possible.
```

Comment:
dev/ops == devops

Reply:
We want to encourage the usage of Devops tooling such as config management, containerisation and automation. This will improve the ease of use for all and increase code reuse amongst the community ultimately leading to an easier time for all.

Reply:
See above, I'd prefer not to mention specific tools by name if possible.